src/plugins/recurring-issues/index.js

```diff
@@ -0,0 +1,227 @@
+
+* (needs volunteers)
+* (needs volunteers)
+
+Please use this issue to document how the release went, any problems during the release, and anything the team might want to know about the release process. This issue should be closed after all patch releases have been completed (or there was no patch release needed).
+
+Resources:
+
+* [Release guidelines](https://eslint.org/docs/maintainer-guide/releases)
+
+`.trim();
+}
+
+/**
+ * Gets the members of a particular team on GitHub
+ * @param {GitHub} github A GitHub API client
+ * @returns {{login: string, name: (string|null)}} A list of team member login names and full names
+ */
+async function getTeamMembers({ github, organizationName, teamName }) {
+
+    /*
+     * NOTE: This will fail if the organization contains more than 100 teams. This isn't
+     * close to being a problem right now, so it hasn't been worth figuring out a good
+     * way to paginate yet, but that would be a good enhancement in the future.
+     */
+    const teams = await github.orgs.getTeams({ org: organizationName, per_page: 100 }).then(res => res.data);
+    const desiredTeam = teams.find(team => team.slug === teamName);
+
+    if (!desiredTeam) {
+        throw new Error(`No team with name ${teamName} found`);
+    }
+
+    const teamMembers = await github.orgs.getTeamMembers({ id: desiredTeam.id, per_page: 100 }).then(res => res.data);
+
+    return Promise.all(teamMembers.map(async member => ({
+        login: member.login,
+        name: await github.users.getById({ id: member.id }).then(res => res.data[0].name)
+    })));
+}
+
+/**
+ * Gets the desired issue body for a release issue, given the date and of the meeting
+ * @param {Moment} meetingDate The date and time when the meeting will take place, as a Moment date
+ * @param {GitHub} github A GitHub API client
+ * @returns {Promise<string>} The text of the issue
+ */
+async function getTscMeetingIssueBody({ meetingDate, github, organizationName, tscTeamName }) {
+    const timeFormatString = "ddd DD-MMM-YYYY HH:mm";
+
+    return `
+
+# Time
+
+UTC ${moment.utc(meetingDate).format(timeFormatString)}:
+- San Francisco: ${moment.tz(meetingDate, "America/Los_Angeles").format(timeFormatString)}
+- New York: ${moment.tz(meetingDate, "America/New_York").format(timeFormatString)}
+- Madrid: ${moment.tz(meetingDate, "Europe/Madrid").format(timeFormatString)}
+- Moscow: ${moment.tz(meetingDate, "Europe/Moscow").format(timeFormatString)}
+- Sydney: ${moment.tz(meetingDate, "Australia/Sydney").format(timeFormatString)}
+- Tokyo: ${moment.tz(meetingDate, "Asia/Tokyo").format(timeFormatString)}
+
+# Location
+
+https://gitter.im/eslint/tsc-meetings
+
+# Agenda
+
+Extracted from:
+
+* Issues and pull requests from the ESLint organization with the ["tsc agenda" label](https://github.com/issues?utf8=%E2%9C%93&q=org%3Aeslint+label%3A%22tsc+agenda%22)
+* Comments on this issue
+
+# Invited
+
+${
+    (await getTeamMembers({
```

Comment:
always better to extract logic into functions with appropriate names for readability. Specially stuff inside template strings.